From a76f71b24510bb37a8b46030937e2c37d68b22d3 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 18 Dec 2023 14:40:49 +0100
Subject: [PATCH 0220/1695] drivers: clk_dt: include missing clk.h header file

clk resources are used in this file. Add missing include.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c04aac6d4a7105aa33bd2c28293e5f11325b2a80 drivers: clk_dt: include missing clk.h header file]
Change-Id: I95861998623e6290a3eb2850801fb61b73088f2b
[gatien: cherry-picked from 23cbf81f2b059ab4c98413ad185d382b42df4a11]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
---
 core/include/drivers/clk_dt.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/include/drivers/clk_dt.h b/core/include/drivers/clk_dt.h
index eb164acb8..5ea10ade8 100644
--- a/core/include/drivers/clk_dt.h
+++ b/core/include/drivers/clk_dt.h
@@ -6,6 +6,7 @@
 #ifndef __DRIVERS_CLK_DT_H
 #define __DRIVERS_CLK_DT_H
 
+#include <drivers/clk.h>
 #include <kernel/dt_driver.h>
 #include <scattered_array.h>
 #include <stdint.h>
-- 
2.47.2

